### ISSUE TITLE
Inject eMob MIT timeseries into pypsa-eur-sec workflow

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#836]
### Tasks

- Similar to #647, make p-e-s use the eMob data from eGon-data.
- Check, if #660 is required for this.
- Use data from `eGon100RE` and `eGon100RE_noflex` scenarios (the latter requires merge of #832).
- If CSV files are usd, they may be exported to `<working_dir>/emobility/` (in the bast case a subdir is created)

The following p-e-s components/vars are relevant, the data can be obtained from the eTraGo tables:
- Load
  - [`p_set`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1317) (keep param `electric_share`): from `EgonPfHvLoadTimeseries.p_set` (sum up), query snippet see **(1)**
- Link
  - [`p_nom`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1328): from `EgonPfHvLink.p_nom` (sum up), query snippet see **(2)**
  - [`p_max_pu`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1337): from `EgonPfHvLinkTimeseries.p_max_pu` (sum up weighted by `p_nom`), query snippet see **(2)**
- Store
  - [`e_nom`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1360) (keep param `options["bev_availability"]` here): from `EgonPfHvStore.e_nom` (sum up), query snippet see **(3)**
  - [`e_min_pu`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1370) (keep simple p-e-s timeseries for now, will change later)

PS: There're some more params that need to be set (e.g. [`electric_share`](https://github.com/openego/pypsa-eur-sec/blob/3db09d4034c08657e90380098505c334ee5ff9f7/scripts/prepare_sector_network.py#L1295)) but we can tackle this later.

### Snippets

**(1)** `EgonPfHvLoadTimeseries.p_set`
```python
with db.session_scope() as session:
    query = session.query(
        EgonPfHvLoad.bus.label("bus_el_id"),
        EgonPfHvLoad.load_id.label("load_id"),
        EgonPfHvLoadTimeseries.p_set,
    ).select_from(
        EgonPfHvLoad,
    ).join(
        EgonPfHvLoadTimeseries,
        EgonPfHvLoadTimeseries.load_id == EgonPfHvLoad.load_id
    ).filter(
        EgonPfHvLoad.scn_name == "eGon100RE_noflex",
        EgonPfHvLoad.carrier == "land transport EV"
    )
charging_load_ts = pd.read_sql(
    query.statement,
    query.session.bind,
    index_col=None
)
```

**(2)** `EgonPfHvLink.p_nom` and `EgonPfHvLinkTimeseries.p_max_pu`
```python
with db.session_scope() as session:
    query = session.query(
        EgonPfHvLink.bus0.label("bus_el_id"),
        EgonPfHvLink.link_id.label("link_id"),
        EgonPfHvLink.p_nom,
        EgonPfHvLinkTimeseries.p_max_pu,
    ).select_from(
        EgonPfHvLink,
    ).join(
        EgonPfHvLinkTimeseries,
        EgonPfHvLinkTimeseries.link_id == EgonPfHvLink.link_id
    ).filter(
        EgonPfHvLink.scn_name == "eGon100RE",
        EgonPfHvLink.carrier == "BEV charger"
    )
charging_load_ts = pd.read_sql(
    query.statement,
    query.session.bind,
    index_col=None
)
charging_load_ts
```

**(3)** `EgonPfHvStore.e_nom`
```python
with db.session_scope() as session:
    query = session.query(
        EgonPfHvStore.bus.label("bus_el_id"),
        EgonPfHvStore.store_id.label("store_id"),
        EgonPfHvStore.e_nom,
    ).filter(
        EgonPfHvStore.scn_name == "eGon100RE",
        EgonPfHvStore.carrier == "battery storage"
    )
storages = pd.read_sql(
    query.statement,
    query.session.bind,
    index_col=None
)
storages
```

**Comment by ClaraBuettner** [comment]
@birgits and @nesnoj: I found this issue concerning the e-Mobility integration into pypsa-eur. Is that still up to date?

**Comment by nesnoj** [comment]
> @birgits and @nesnoj: I found this issue concerning the e-Mobility integration into pypsa-eur. Is that still up to date?

Yes, I think so - eMob MIT has never been integrated into p-e-s for eGon100RE.

**Comment by nesnoj** [comment]
Is this still needed @ClaraBuettner ?

**Comment by nesnoj** [comment]
Outdated
